ONSdigital/takeon-business-layer
# Changed log level from debug to info

# Description
Changed the log level of BL from DEBUG to INFO

# How to test?

- Go to any environment run the 'Search Contributor' and 'View Form' 
- Go to AWS Management Console -> Cloudwatch -> Logs -> LogGroups and search for 'ecs/spp-es-takeon-<ENV>-business-layer' and verify the level of log
